README.md

```diff
@@ -42,7 +42,15 @@ wget \
   -O /usr/local/bin/hzcloud && chmod +x /usr/local/bin/hzcloud
 ```
 ## Authentication with Hazelcast Cloud
-After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer).
+After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer). 
+
+`hzcloud` is configured against [Hazelcast Cloud](https://cloud.hazelcast.com) by default. However, you can use it with [Hazelcast Viridian](https://viridian.hazelcast.com/) as well by setting `HZ_CLOUD_API_URL` environment variable:  
```

Comment:
As much as I think the word `against` is correct in this context, I still can imagine a situation where it be confusing for someone.
Maybe let's say something like `configured to use [Hazelcast Cloud]`?